server/test/factories.ts

```diff
@@ -444,6 +444,9 @@ export async function buildAttachment(overrides: Partial<Attachment> = {}) {
     updatedAt: new Date("2018-01-02T00:00:00.000Z"),
     ...overrides,
   });
+
+  attachment.key = `public/${overrides.userId}/${attachment.id}/images.docx`;
```

Comment:
Not loving that `images.docx` is hardcoded here, we should also remove `key` on L439 if it's just going to be overridden anyway.

Allow `buildAttachment` to accept a second argument with optional file name parameter?

Reply:
Sure, makes more sense.

Reply:
Can't skip `key` though. It's not nullable.